components/schema-migrator/migrations/director/20210413140518_update-runtime-tenantId-update-cascade.down.sql

```diff
@@ -0,0 +1,23 @@
+BEGIN;
```

Comment:
These tables also have a relation to the `runtimes` table:
- `api_runtime_auths`
- `webhooks`